airflow/www/extensions/init_views.py

```diff
@@ -323,11 +316,20 @@ def init_api_experimental(app):
     app.extensions["csrf"].exempt(endpoints.api_experimental)
 
 
-def init_api_auth_provider(app):
+def init_api_auth_manager(connexion_app: connexion.FlaskApp):
     """Initialize the API offered by the auth manager."""
     auth_mgr = get_auth_manager()
-    blueprint = auth_mgr.get_api_endpoints()
-    if blueprint:
-        base_paths.append(blueprint.url_prefix)
-        app.register_blueprint(blueprint)
-        app.extensions["csrf"].exempt(blueprint)
+    auth_mgr.set_api_endpoints(connexion_app)
+
+
+def init_cors_middleware(connexion_app: connexion.FlaskApp):
```

Comment:
@RobbeSneyders - followint your advice we replaced our custom CORS implementation with the middleware - does it look right?